submissions/ses3332002/tiny-js-world-oop/index.js

```diff
@@ -0,0 +1,81 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+
+class Animal {
+  constructor (name, friends, gender, saying, species) {
+    this.name = name;
+    this.friends = friends;
+    this.gender = gender;
+    this.saying = saying;
+    this.species = species;
+    this.legs = 4;
```

Comment:
There are no good reasons to have a default value here.
Four-legged animals are no better than e.g. humans.
There is a good reason the e.g. `species` prop doesn't have any default value. So why treating legs in a different way?

Reply:
I remade it.